content/en/docs/develop/porting.md

```diff
@@ -41,44 +41,43 @@ For this tutorial, we will be targeting the network utility program [`iperf3`](h
- * It is a standalone tool which does one thing;
- * It's [GNU Make](https://www.gnu.org/software/make/) and C-based;
- * And it's quite useful :)
+1. It can run as a "server-type" application, receiving and processing requests for clients;
+1. It is a standalone tool which does one thing;
+1. It's [GNU Make](https://www.gnu.org/software/make/) and C-based;
+1. And it's quite useful :)
 
 Understanding some aspects of how an application works, particularly how it is built, is required when bringing it to Unikraft.
 Usually during the porting process we also end up diving through the source code, and in the worst-case scenario, have to make a change to it.
 More on this is covered in the [Patching](./docs/develop/patching) section.
 
 Let's start by simply trying to follow the steps to compile the application from source.
 
-
 ### Compiling the Application from Source
 
 The [`README`](https://github.com/esnet/iperf/blob/master/README.md) for the `iperf3` program has relatively simple build instructions, and uses GNU Make which is a first good sign.
 Unikraft uses GNU Make to handle its internal builds and so when we see an application using Make, it makes porting a little easier.
-For non-Make type build systems, such as CMake, Bazel, etc., it is still possible to bring this application to Unikraft, but the flags, files, and compile-time options, etc. will have to be considered with more care as they do not necessarily align in the same ways.
-It is still possible to bring an application using an alternative build system, but you must closely follow how the program is built in order to bring it to Unikraft.
+For non-Make type build tools (such as CMake, Bazel, etc.) it is still possible to bring this application to Unikraft, but the flags, files, and compile-time options and other items will have to be considered with more care as they do not necessarily align in the same ways.
+It is still possible to bring an application using an alternative build tool, but you must closely follow how the program is built-in order to bring it to Unikraft.
 
 Let's walk through the build process of `iperf3` from its `README`:
 
 1. First we obtain the source code of the application:
 
    ```bash
-   $ git clone https://github.com/esnet/iperf.git
+   git clone https://github.com/esnet/iperf.git
    ```
 
-2. Then, we are asked to configure and build the application:
+1. Then, we are asked to configure and build the application:
 
    ```bash
-   $ cd ./iperf
-   $ ./configure;
-   $ make
+   cd ./iperf
+   ./configure;
+   make
    ```
 
 If this has worked for you, your terminal will be greeted with several pieces of useful information:
 
-1. The first thing we did was run `./configure`: an auto-generated utility program part of the [`automake`](https://www.gnu.org/software/automake/) build system.
+1. The first thing we did was run `./configure`: an auto-generated utility program part of the [`automake`](https://www.gnu.org/software/automake/) build tool.
```

Comment:
You're right, it is too large for a bullet item. I would just remove the bullet since I don't think it gives enough information for a subsection and it looks fine if it just flows as a new paragraph.